include/geode/mesh/io/edged_curve_input.h

```diff
@@ -38,12 +38,16 @@ namespace geode
     /*!
      * API function for loading an EdgedCurve.
      * The adequate loader is called depending on the filename extension.
-     * @param[out] edged_curve Loaded EdgedCurve.
+     * @param[in] type Data structure type.
      * @param[in] filename Path to the file to load.
      */
     template < index_t dimension >
-    void load_edged_curve(
-        EdgedCurve< dimension >& edged_curve, absl::string_view filename );
+    std::unique_ptr< EdgedCurve< dimension > > load_edged_curve(
+        const MeshType& type, absl::string_view filename );
+
+    template < index_t dimension >
```

Comment:
Add doc for the second function? For all mesh types

Reply:
Is it necessary?

Reply:
Ad you wish